internal/proposals/migration-hooks/migration-hooks-proposal.md

```diff
@@ -141,14 +134,15 @@ package jobManager
 register(job1)
 type job1 struct{}
 
-func (j job1) execute() {
+func (j job1) execute(cfg *config.Config, kubeClient kubernetes.Interface) {
```

Comment:
I think we also seed cancellation context here. But maybe it's too technical/low-level to introduce in the proposal?